compose/config/serialize.py

```diff
@@ -45,7 +46,7 @@ def denormalize_config(config):
         if 'external_name' in vol_conf:
             del vol_conf['external_name']
 
-    if config.version in (V3_1,):
+    if config.version in (V3_1, V3_2):
```

Comment:
Can we change this to `< V3_1` for the same reason? We don't want to have to remember to add to this list every time we change a version.

Maybe we should go even further and have a function like `version_supports_secrets(version)`, and use it both here and in `compose/config/config.py` ?